pyan/writers.py

```diff
@@ -234,127 +208,115 @@ def run(self):
         self.grouped = graph.grouped
         self.indent_level = 0
         self.edge_id = 0
 
     def start_graph(self):
         self.write('<?xml version="1.0" encoding="UTF-8" standalone="no"?>')
         self.write(
-                '<graphml xmlns="http://graphml.graphdrawing.org/xmlns"'
-                ' xmlns:java='
-                '"http://www.yworks.com/xml/yfiles-common/1.0/java"'
-                ' xmlns:sys='
-                '"http://www.yworks.com/xml/yfiles-common/markup/primitives'
-                '/2.0" xmlns:x="http://www.yworks.com/xml/yfiles-common/'
-                'markup/2.0" xmlns:xsi="http://www.w3.org/2001/'
-                'XMLSchema-instance" xmlns:y="http://www.yworks.com/xml/'
-                'graphml" xmlns:yed="http://www.yworks.com/xml/yed/3"'
-                ' xsi:schemaLocation="http://graphml.graphdrawing.org/xmlns'
-                ' http://www.yworks.com/xml/schema/graphml/1.1/'
-                'ygraphml.xsd">')
+            '<graphml xmlns="http://graphml.graphdrawing.org/xmlns"'
+            " xmlns:java="
+            '"http://www.yworks.com/xml/yfiles-common/1.0/java"'
+            " xmlns:sys="
+            '"http://www.yworks.com/xml/yfiles-common/markup/primitives'
+            '/2.0" xmlns:x="http://www.yworks.com/xml/yfiles-common/'
+            'markup/2.0" xmlns:xsi="http://www.w3.org/2001/'
+            'XMLSchema-instance" xmlns:y="http://www.yworks.com/xml/'
+            'graphml" xmlns:yed="http://www.yworks.com/xml/yed/3"'
+            ' xsi:schemaLocation="http://graphml.graphdrawing.org/xmlns'
+            " http://www.yworks.com/xml/schema/graphml/1.1/"
+            'ygraphml.xsd">'
+        )
         self.indent()
         self.write('<key for="node" id="d0" yfiles.type="nodegraphics"/>')
         self.write('<key for="edge" id="d1" yfiles.type="edgegraphics"/>')
         self.write('<graph edgedefault="directed" id="Graph">')
         self.indent()
 
     def start_subgraph(self, graph):
-        self.log('Start subgraph %s' % graph.label)
+        self.log("Start subgraph %s" % graph.label)
 
         self.write('<node id="%s:" yfiles.foldertype="group">' % graph.id)
         self.indent()
         self.write('<data key="d0">')
         self.indent()
-        self.write('<y:ProxyAutoBoundsNode>')
+        self.write("<y:ProxyAutoBoundsNode>")
         self.indent()
         self.write('<y:Realizers active="0">')
         self.indent()
-        self.write('<y:GroupNode>')
+        self.write("<y:GroupNode>")
         self.indent()
         self.write('<y:Fill color="#CCCCCC" transparent="false"/>')
-        self.write('<y:NodeLabel modelName="internal" modelPosition="t" '
-                   'alignment="right">%s</y:NodeLabel>'
-                   % graph.label)
+        self.write(
+            '<y:NodeLabel modelName="internal" modelPosition="t" ' 'alignment="right">%s</y:NodeLabel>' % graph.label
```

Comment:
Auto-format borkage. The string snippets could be combined (see the `' '` in the middle).

Reply:
fixed them

Reply:
Thanks!